compactor2/tests/layouts/core.rs

```diff
@@ -352,6 +365,8 @@ async fn l1_with_non_overlapping_l0_larger() {
                     .with_min_time(50 + i * 50)
                     .with_max_time(99 + i * 50)
                     .with_compaction_level(CompactionLevel::FileNonOverlapped)
+                    // L1 files with smaller max_l0_crearted_at to indicate they inlcude data created before the L0 files
```

Comment:
```suggestion
                    // L1 files with smaller max_l0_created_at to indicate they include data created before the L0 files
```

a similar typo is copied several places

Reply:
Fixed all places